inspirehep/utils/record.py

```diff
@@ -31,3 +35,53 @@ def get_title(record):
             return title['title']
 
     return title
+
+
+def get_smart_value(record, key, default=None):
```

Comment:
I know it's silly, but to me seems this function will return me a *smart value* rather than being smart in returning values :smile: 

Wonder if it's better called: `smart_get_value()` (although it makes it a bit hidden that is a getter) or `get_value_smart()`.